plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/core/CoreResources_uk_UA.properties

```diff
@@ -0,0 +1,595 @@
+action_menu_transaction_pref_page_link = \u0414\u0438\u0432. <a>"{0}"</a> \u0434\u043B\u044F \u043D\u0430\u043B\u0430\u0448\u0442\u0443\u0432\u0430\u043D\u044C \u0442\u0440\u0430\u043D\u0437\u0430\u043A\u0446\u0456\u0439, \u043F\u043E\u0432\u2019\u044F\u0437\u0430\u043D\u0438\u0445 \u0437 \u0442\u0438\u043F\u0430\u043C\u0438 \u043F\u0456\u0434\u043A\u043B\u044E\u0447\u0435\u043D\u043D\u044F.
+action_menu_transaction_pref_page_link_extended = \u0414\u0438\u0432. <a>"{0}"</a> \u0434\u043B\u044F \u043D\u0430\u043B\u0430\u0448\u0442\u0443\u0432\u0430\u043D\u044C \u0442\u0440\u0430\u043D\u0437\u0430\u043A\u0446\u0456\u0439, \u043F\u043E\u0432\u2019\u044F\u0437\u0430\u043D\u0438\u0445 \u0437 \u0442\u0438\u043F\u0430\u043C\u0438 \u043F\u0456\u0434\u043A\u043B\u044E\u0447\u0435\u043D\u043D\u044F.\n\u041D\u0430\u043B\u0430\u0448\u0442\u0443\u0432\u0430\u043D\u043D\u044F \u0434\u0436\u0435\u0440\u0435\u043B\u0430 \u0434\u0430\u043D\u0438\u0445 \u043C\u0430\u044E\u0442\u044C \u043F\u0435\u0440\u0435\u0432\u0430\u0433\u0443 \u043D\u0430\u0434 \u0456\u043D\u0448\u0438\u043C\u0438 \u043D\u0430\u043B\u0430\u0448\u0442\u0443\u0432\u0430\u043D\u043D\u044F\u043C\u0438.
+
+pref_page_connection_types_label_auto_close_enabled_tip = \u0417\u0430\u043A\u0440\u0438\u0432\u0430\u0442\u0438 \u0442\u0440\u0438\u0432\u0430\u043B\u0456 \u0442\u0440\u0430\u043D\u0437\u0430\u043A\u0446\u0456\u0457 \u043F\u0456\u0441\u043B\u044F \u043F\u0435\u0440\u0456\u043E\u0434\u0443 \u043D\u0435\u0430\u043A\u0442\u0438\u0432\u043D\u043E\u0441\u0442\u0456 (\u0431\u0435\u0437 \u0434\u0456\u0439 \u043A\u043E\u0440\u0438\u0441\u0442\u0443\u0432\u0430\u0447\u0430).\n\u0412\u0456\u0434\u043A\u0430\u0442 \u0431\u0443\u0434\u0435 \u0432\u0438\u043A\u043E\u043D\u0430\u043D\u043E \u043F\u0435\u0440\u0435\u0434 \u0437\u0430\u0432\u0435\u0440\u0448\u0435\u043D\u043D\u044F\u043C \u0442\u0440\u0430\u043D\u0437\u0430\u043A\u0446\u0456\u0457.\n\u0417\u043D\u0430\u0447\u0435\u043D\u043D\u044F \u0437\u0430 \u0437\u0430\u043C\u043E\u0432\u0447\u0443\u0432\u0430\u043D\u043D\u044F\u043C \u0434\u043B\u044F \u043E\u0431\u0440\u0430\u043D\u043E\u0433\u043E \u0442\u0438\u043F\u0443 \u043F\u0456\u0434\u043A\u043B\u044E\u0447\u0435\u043D\u043D\u044F.
+pref_page_connection_types_label_auto_close_ttl_tip = \u0427\u0430\u0441 \u0456\u0441\u043D\u0443\u0432\u0430\u043D\u043D\u044F \u0442\u0440\u0430\u043D\u0437\u0430\u043A\u0446\u0456\u0457 \u0432 \u0440\u0435\u0436\u0438\u043C\u0456 \u043E\u0447\u0456\u043A\u0443\u0432\u0430\u043D\u043D\u044F (\u0443 \u0441\u0435\u043A\u0443\u043D\u0434\u0430\u0445).
+
+action_menu_tools_find_in_navigator = \u0417\u043D\u0430\u0439\u0442\u0438 \u0432 \u041D\u0430\u0432\u0456\u0433\u0430\u0442\u043E\u0440\u0456
```

Comment:
The Navigator word can start from the small "н", because only in English we are trying to use all nouns from capital letters, but not in Russian or Ukrainian.

Reply:
I fixed it, there really should have been a lowercase letter.

I am asking you very much, do not compare us with russia. They try us to kill

Reply:
I'm talking about language. It is harmless. 

Thanks for the changes :)